### ISSUE TITLE
Question: will the app work without prod.keys?

### ISSUE THREAD

**Issue by 1DontEx1st** [ivaano/NsxLibraryManager#38]
I've seen other utilities that can pull info from dumped files without needing the keys, so I'm wondering if the reason my library is currently empty is because I haven't copied my file over yet.

**Comment by ivaano** [comment]
No, keys are needed to read the file contents, as we are using https://github.com/Thealexbarney/LibHac to get the file info.
